src/xcode/ENA/ENA/Source/Client/HTTP Client/HTTPClient.swift

```diff
@@ -1492,7 +1492,38 @@ private extension URLRequest {
 	
 		return request
 	}
-	
+
+	static func DSCListRequest(
```

Comment:
```suggestion
	static func requestDSCList(
```

Reply:
I changed it to `dscListRequest` to keep is consistent. Request is always at the ende if the static func creating the requests.